packages/react/npm/jsx-dev-runtime.react-server.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+if (process.env.NODE_ENV === 'production') {
+  module.exports = require('./cjs/react-jsx-dev-runtime.react-server.production.min.js');
```

Comment:
Just spotted this. We no longer ship minified so I'll need to follow-up on this one: https://github.com/facebook/react/pull/28939